contributors.json

```diff
@@ -0,0 +1,80 @@
+   "Jeff Klukas" : "https://github.com/jklukas",
+   "Jeremy Whiting" : "https://github.com/whitingjr",
+   "Jesper Pedersen" : "https://github.com/jesperpedersen",
+   "Joe Kutner" : "https://github.com/jkutner",
+   "Jordan Lewis" : "https://github.com/jordanlewis",
+   "Jorge Solorzano" : "https://github.com/jorsol",
+   "KimBisgaardDmi" : "https://github.com/KimBisgaardDmi",
+   "Laurenz Albe" : "https://github.com/laurenz",
+   "Magnus" : "https://github.com/magJ",
+   "Magnus Hagander" : "https://github.com/mhagander",
+   "Marc Petzold" : "https://github.com/dosimeta",
+   "Marc Slemko" : "https://github.com/znep",
+   "Marios Trivyzas" : "https://github.com/matriv",
+   "Mathias Fußenegger" : "https://github.com/mfussenegger",
+   "Michael Glaesemann" : "https://github.com/grzm",
+   "MichaelZg" : "https://github.com/michaelzg",
+   "Michele Mancioppi" : "https://github.com/michele-mancioppi",
+   "Minglei Tu" : "https://github.com/tminglei",
+   "Pavel Raiskup" : "https://github.com/praiskup",
+   "Pawel" : "https://github.com/veselov",
+   "Petro Semeniuk" : "https://github.com/PetroSemeniuk",
+   "Philippe Marschall" : "https://github.com/marschall",
+   "Piyush Sharma" : "https://github.com/ps-sp",
+   "Rikard Pavelic" : "https://github.com/zapov",
+   "Robert 'Bobby' Zenz" : "https://github.com/RobertZenz",
+   "Robert Zenz" : "https://github.com/RobertZenz",
+   "Roman Ivanov" : "https://github.com/romani",
+   "Sebastian Utz" : "https://github.com/seut",
+   "Sehrope Sarkuni" : "https://github.com/sehrope",
+   "Selene Feigl" : "https://github.com/sfeigl",
+   "Sidi Mohamed EL AATIFI" : "https://github.com/elaatifi",
+   "Simon Stelling" : "https://github.com/stellingsimon",
+   "Stephen Nelson" : "https://github.com/lordnelson",
+   "Steve Ungerer" : "https://github.com/scubasau",
+   "Tanya Gordeeva" : "https://github.com/tmgordeeva",
+   "Thach Hoang" : "https://github.com/thachhoang",
+   "Trygve Laugstøl" : "https://github.com/trygvis",
+   "Vladimir Gordiychuk" : "https://github.com/Gordiychuk",
+   "Vladimir Sitnikov" : "https://github.com/vlsi",
+   "Zemian Deng" : "https://github.com/zemian",
+   "aryabukhin" : "https://github.com/aryabukhin",
+   "bazzargh" : "https://github.com/bazzargh",
+   "bd-infor" : "https://github.com/bd-infor",
+   "benbenw" : "https://github.com/benbenw",
+   "benoit" : "https://github.com/benbenw",
+   "bpd0018" : "https://github.com/bpd0018",
+   "chalda" : "https://github.com/ochaloup",
+   "djydewang" : "https://github.com/djydewang",
+   "eperez" : "https://github.com/eperez",
+   "goeland86" : "https://github.com/goeland86",
+   "mjanczykowski" : "https://github.com/mjanczykowski",
+   "rnveach" : "https://github.com/rnveach",
+   "slmsbrhgn" : "https://github.com/slmsbrhgn",
+   "trtrmitya" : "https://github.com/trtrmitya",
+   "zapov" : "https://github.com/zapov",
+   "Álvaro Hernández Tortosa" : "https://github.com/ahachete",
+   "Ãtienne BERSAC" : "https://github.com/bersace"
```

Comment:
Apparently there is utf-8 kind of issue with release_notes_filter.pl to be fixed, however the rest should be fine